installation/sql/mysql/extensions.sql

```diff
@@ -882,3 +882,93 @@ CREATE TABLE IF NOT EXISTS `#__action_logs_users` (
   PRIMARY KEY (`user_id`),
   KEY `idx_notify` (`notify`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 DEFAULT COLLATE=utf8mb4_unicode_ci;
+
+-- --------------------------------------------------------
+
+--
+-- Table structure for table `#__guidedtours`
+--
+
+CREATE TABLE IF NOT EXISTS `#__guidedtours` (
+  `id` int NOT NULL AUTO_INCREMENT,
+  `asset_id` int DEFAULT 0,
+  `title` varchar(255) NOT NULL,
+  `description` text NOT NULL,
+  `ordering` int NOT NULL DEFAULT 0,
+  `extensions` text NOT NULL,
+  `url` varchar(255) NOT NULL,
+  `overlay` tinyint NOT NULL  DEFAULT 0,
+  `created` datetime NOT NULL,
+  `created_by` int NOT NULL DEFAULT 0,
+  `modified` datetime NOT NULL,
+  `modified_by` int NOT NULL DEFAULT 0,
+  `checked_out_time` datetime NOT NULL,
+  `checked_out` int NOT NULL DEFAULT 0,
+  `published` tinyint NOT NULL DEFAULT 0,
+  `state` tinyint NOT NULL DEFAULT '1',
+  PRIMARY KEY (`id`),
+  KEY `idx_asset_id` (`asset_id`),
+  KEY `idx_title` (`title`(191)),
+  KEY `idx_created` (`created`),
+  KEY `idx_created_by` (`created_by`),
+  KEY `idx_modified` (`modified`),
+  KEY `idx_modified_by` (`modified_by`),
+  KEY `idx_checked_out` (`checked_out`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 DEFAULT COLLATE=utf8mb4_unicode_ci;
+
+--
+-- Dumping data for table `#__guidedtours`
+--
+
+INSERT INTO `#__guidedtours` (`id`, `asset_id`, `title`,`description`, `ordering`, `extensions`, `url`,`overlay`, `created`, `created_by`, `modified`, `modified_by`, `checked_out_time`, `checked_out`, `published`, `state`) VALUES
+(1, 91, 'How to create a Guided Tour in Joomla Backend?','<p>This Tour will show you that how you can create Guided Tour in Joomla Backend!</p>', 0, '[\"com_guidedtours\"]', 'http://localhost/soc21_guided-tour/administrator/index.php?option=com_guidedtours',1, '2021-09-17 18:33:56', 430, '2021-09-17 18:33:56', 430, '2021-09-17 18:46:24', 430, 1, 1);
```

Comment:
@bembelimen @shivamdiehard - I'm going to admit I don't know the specifics of optimising some of the SQL here, but I know @jatinS-dev has some fatal flaws in the statements as part of this review request. Can you please correct some of my statements.

```INSERT INTO `#__guidedtours` (`id`, `asset_id`, `title`,`description`, `ordering`, `extensions`, `url`,`overlay`, `created`, `created_by`, `modified`, `modified_by`, `checked_out_time`, `checked_out`, `published`, `state`) VALUES
(0, 0, 'How to create a Guided Tour in Joomla Backend?','<p>This Tour will show you that how you can create Guided Tour in Joomla Backend!</p>', 0, '[\"com_guidedtours\"]', 'administrator/index.php?option=com_guidedtours',1, '', 0, '', 0, '', 0, 1, 1);```

- id should not be set in this file. It will automatically be set by the autoincrement on the field
- asset_id - this should also be set by Joomla, not the installer, as there may be an asset somewhere else with asset_id=91. _@bembelimen I don't know enough about the asset_id field's relationship to #__assets to know if it's implemented correctly or needed for com_guidedtour_
- title ✔️ 
- description ✔️ 
- ordering ✔️
- extensions ✔️
- url should not be an absolute url. Make URLS relative URLS.- it will break on sites if it's set to http://localhost/soc21_guided-tour/ Also look at the code for where you have the URL field in the tour and steps pages - relative urls means it will always apply to the site that the tour is executed on. Absolute URLS mean that the tours are not as portable.
- overlay ✔️
- `created`, `created_by`, `modified`, `modified_by`, `checked_out_time`, `checked_out` - All 6 of these fields should be set to '' (for dates) or 0 (for the User ID) as the dates are not consequential, and your user ID of 430 may not exist in installations. _@bembelimen again, deferring to you if this is correct. Alternate for created date field would to have it load current date in the SQL_
- published ✔️
- state ✔️

Reply:
This would also have slipped through until a later code review except that I noticed that the installation had the tour checked out on install. So resetting checked_out_time and checked_out will resolve that issue.